engine-dmn/feel-juel/pom.xml

```diff
@@ -12,7 +12,7 @@
   <name>camunda DMN - engine FEEL - JUEL</name>
 
   <properties>
-    <!-- We shade artifacts into the jar, so we need to generate 
+    <!-- We shade artifacts into the jar, so we need to generate
     a dependency BOM for the license book -->
     <skip-third-party-bom>false</skip-third-party-bom>
```

Comment:
We can skip the third party bom now, yes. We need the exclude in clirr, otherwise it will report some missing classes now (that previously existed at the shaded path).